docs/userguide/storagedriver/overlayfs-driver.md

```diff
@@ -133,13 +148,85 @@ You can verify all of these constructs from the output of the `mount` command.
 (Ellipses and line breaks are used in the output below to enhance readability.)
 
     $ mount | grep overlay
-    overlay on /var/lib/docker/overlay/73de7176c223.../merged
-    type overlay (rw,relatime,lowerdir=/var/lib/docker/overlay/1d073211c498.../root,
-    upperdir=/var/lib/docker/overlay/73de7176c223.../upper,
-    workdir=/var/lib/docker/overlay/73de7176c223.../work)
+    overlay on /var/lib/docker/overlay/ec444863a55a.../merged
+    type overlay (rw,relatime,lowerdir=/var/lib/docker/overlay/55f1e14c361b.../root,
+    upperdir=/var/lib/docker/overlay/ec444863a55a.../upper,
+    workdir=/var/lib/docker/overlay/ec444863a55a.../work)
 
 The output reflects that the overlay is mounted as read-write ("rw").
 
+
+## Image layering and sharing with OverlayFS (`overlay2`)
+
+While the `overlay` driver only works with a single lower OverlayFS layer and
+hence requires hard links for implementation of multi-layered images, the
+`overlay2` driver natively supports multiple lower OverlayFS layers (up to 128).
+
+The `overlay2` driver consumes less inodes than the `overlay` driver.
```

Comment:
Maybe also a mention about overlay2 offering significantly better performance for commands that create new layes, such as `docker build`, `docker commit`, and `docker pull`